sdks/go/pkg/beam/core/runtime/exec/datasource.go

```diff
@@ -134,7 +134,7 @@ func (n *DataSource) Process(ctx context.Context) error {
 		}
 		pe.Timestamp = t
 		pe.Windows = ws
-
+		pe.Pane = pn
```

Comment:
Please keep the blank line before the next block. 
Blank lines used carefully help readability.